cppa_slack_transcript_tracker/tests/test_workspace.py

```
@@ -0,0 +1,46 @@
+    get_workspace_root,
+    get_data_dir,
+    set_working_directory,
+)
+
+
+def test_get_workspace_root_returns_path_from_config():
+    """get_workspace_root returns config workspace path for this app slug."""
+    with patch(
+        "cppa_slack_transcript_tracker.workspace.get_workspace_path"
+    ) as mock_get:
+        mock_get.return_value = Path("/tmp/workspace/cppa_slack_transcript_tracker")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/workspace/cppa_slack_transcript_tracker"

(S108)